src/language/dafnyInstallation.ts

```diff
@@ -15,17 +15,24 @@ import Configuration from '../configuration';
 const ArchiveFileName = 'dafny.zip';
 const mkdirAsync = promisify(fs.mkdir);
 
+// If someone deletes a configured language server runtime or compiler path, there will remain an empty string.
+// Previously, it would have been interpreted wrongly like a "relative path".
+// Now it is interpreted as "switch back to default"
+function ifNullOrEmpty(a: string | null, b: string) {
```

Comment:
This seems like a strange function because `null` and `''` are both falsy in JavaScript, so instead of `isNullOrEmpty(a,b)`, I think you could also do `a || b`

Reply:
Good point. I'll even inline a || b since it's so simple.